include/spaND.h

```diff
@@ -43,14 +57,16 @@ inline std::string part2str(PartKind pk) {
            "ErrorUnknownPart"));    
 };
 
-typedef std::unique_ptr<Eigen::MatrixXd> pMatrixXd;
-typedef std::unique_ptr<Eigen::VectorXd> pVectorXd;
-typedef std::unique_ptr<Eigen::VectorXi> pVectorXi;
+typedef std::unique_ptr<Eigen::MatrixXd>   pMatrixXd;
```

Comment:
Maybe `using` instead of `typedef`, while we're at it?

Reply:
Good idea! I'll mark this as resolved once I make the change.